src/Blocks/BlockAnvil.h

```diff
@@ -10,6 +10,11 @@
 
 
 
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnon-virtual-dtor"
+#endif
+
```

Comment:
I'm going to try tagging all the handlers final, then give the base a =default protected destructor. Hopefully silences it